src/backend/mpi/swdsm.cpp

```diff
@@ -1044,15 +1044,15 @@ void set_sighandler(){
 	sigaction(SIGSEGV, &sa, &old_sigaction);
 }
 
-void argo_initialize(unsigned long long size){
+void argo_initialize(unsigned long long argo_size, size_t cache_size){
```

Comment:
Should we use this opportunity to change this from unsigned long long to size_t?

Reply:
addressed in d762963